content/posts/blog/2024-01-22-oci-in-2024.md

```diff
@@ -0,0 +1,35 @@
+---
+title: OCI in 2024 and TOB Election Results
+author:
+  name: Open Container Initiative
+  tag: oci
+tags: blog
+date: 2024-01-22
+---
+
+The OCI wrapped up a great 2023 and shipped some new releases and initatives:
+
+* Shipped OCI Runtime Spec 1.1: https://opencontainers.org/release-notices/v1-1-0-runtime-spec/ 
+* Established Image Compatibility Working Group: https://github.com/opencontainers/tob/pull/128
+* Established FreeBSD Working Group: https://github.com/opencontainers/tob/pull/133
+
+We also started off the new year by shipping runc v1.11:
+https://github.com/opencontainers/runc/releases/tag/v1.1.11
+
+To further kick off 2024, we are excited to announce the following four board members are joining or returning to the [OCI Technical Oversight Board (TOB)](https://opencontainers.org/about/tob). The TOB comprises independently elected individuals who provide oversight of the technical leadership and serve as a point of appeal. Each member has been elected to serve a two-year term:
+
+* TODO
+
+These TOB members join the following existing members, who are each in the middle of two-year terms:
+
+* Josh Dolitsky [Chainguard]
+* Phil Estes [Amazon]
+* Jon Johnson [Google]
+* Samuel Karp [Google]
+* Nisha Kumar [Oracle]
```

Comment:
This is the list of members who's seats are up for election. Here's the list with terms expiring in 2025:

```suggestion
* Brandon Mitchell [Independent]
* Vincent Batts [Microsoft]
* Derek McGowan [Docker]
* Aleksa Sarai [SUSE]
```